daffodil-test/src/test/java/org/goodudfs/example/StringFunctions/Replace.java

```diff
@@ -0,0 +1,21 @@
+package org.goodudfs.example.StringFunctions;
+
+import java.io.Serializable;
+
+import org.apache.daffodil.udf.FunctionClassInfo;
+
+@FunctionClassInfo(
+		name = "replace",
+		namespace = "com.ext.goodudfs.StringFunctions"
```

Comment:
Do these have to be strings? Can we get the package name off the class somehow, so that maintaining these things is automated?

Reply:
I suggest namespaceURI as the name of the member